refs/lexicon/RFC-Lexicon-2.md

```diff
@@ -0,0 +1,126 @@
+
+It is composed of one or more [Data Capture Fragments](#data-capture-fragment).
+
+It is obtained in any way, including: directly submitted by a [Submitter](#submitter)(who is the same as the Data Subject, or not), transferred from System to System, generated or inferred by a System.
+
+### Data Capture Fragment
+
+Part of a [Data Capture](#data-capture). A meaningful compound of atomic information units (e.g, data fields of different type, files, and to them associated labels) that are meaningfully captured together.
+
+Examples of situations in which information units are meaningfully captured together include:
+- splitting them would introduce an obstacle to making sense of the information that they carry. For example, elements of an address such as street number and street name,
+- it doesn’t make sense to capture one of them today and the other tomorrow,
+- they are not likely to be modified separately.
+
+Also know as Capture Fragment, or Fragment.
+
+### Data Consumer
+
+A human, or a other entity consuming the data concerning [Data Subjects](#data-subject).
+
+### Data Subject
+
+A human, concerned by particular data that a [System](#system) is processing. Holder of rights over the data concerning them.
+
+### DPO
+
+A Data Protection Officer. An individual or a company charged with responding to [Privacy Requests](#privacy-request) in the name of an [Organization](#organization).
+
+### Fragment
+
+See [Data Capture Fragment](#data-capture-fragment).
+
+### GDPR
+
+General Data Protection Regulation. A regulation in the European Union. See [official text](https://eur-lex.europa.eu/eli/reg/2016/679/oj).
+
+### HIPPA
+
+Health Insurance Portability and Accountability Act. A regulation in the USA. See [official text](https://www.govinfo.gov/content/pkg/PLAW-104publ191/html/PLAW-104publ191.htm).
+
+### Internet User
+
+See [User](#user).
+
+### Organization
+
+A structure, commercial or not, processing [Data Subjects'](#data-subject) data as a part of its operations.
+Also known as "controller", "data controller", and "processor" under [GDPR](#GDPR).
+Also know as "business" under [CCPA](#CCPA) and [CPRA](#CPRA).
+
+It operates one or more [Systems](#system).
+
+### Privateform
+
+[User](#user)-facing software that uses [blindnet devkit](#blindnet-devkit) to capture user data (and to it associated metadata) in a way that restitutes confidentiality and control to the [Data Subject](#data-subject).
+
+### Privacy Request
+
+A request made by a [Data Subject](#data-subject) in order to regulate privacy in the relationship they have with a [System](#system) or an [Organization](#organization).
+
+Privacy Requests concern any aspect of the [privacy-enabled connectedness](../notion-of-privacy/notion-of-privacy.md#privacy-enabled-connectedness), including transparency, confidentiality and control over data processing.
+
+### System
```

Comment:
Rephrase to e.g.

A logical unit of the privacy layer provided by the blindnet devkit for processing of [Data Subjects'](#data-subject) data.

Reply:
I am not sure this would introduce more clarity as "logical unit" is not very well defined, and neither is "privacy layer", and in fact one can consider the Data Capture to be a logical unit and not a Privacy Request.

Reply:
In HLC and HLA documents, the term System refers to a client's system. From the definition here, I assumed it refers to combination of blindnet devkit components (which is called **our system** in the docs). 

We need both definitions - for client's system and our system.
**Our system** needs a better name (not to be confused with client systems).

For the definition of **our system**, I propose:
_Combination of blindnet components, integrated via the blindnet devkit into a client's system, that work as the unit of processing of Data Subject's data._
_A single (our) system can be used by one client or multiple clients. A client can use multiple (our) systems._
_Certain components of a (our) system may interoperate with other (our) systems while maintaining autonomy._

For client's system, something like:
_A combination of one or more interconnected (architectural?) components (servers, UIs, databases, etc.)._
_Components of a client's system can integrate the blindnet devkit components in order to provide various privacy features to their users._

Reply:
@m4rk055 "Our System" is now "blincnet devkit" - I've change it in HLC and HLA. If some remain, they should be updated.

System now means  the Client System.

I think a broader definition of the System is better, as we may, trying to reach precision, ommit certain configurations that we want to consider Systems. Also I am trying to avoid defining any term by introducing other terms like 'architectural components' that would require futher definition.